tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "forceConsistentCasingInFileNames": true,
     "removeComments": true,
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "types": ["jest"]
```

Comment:
will this exclude every other library?

Reply:
This will exclude every other `@type/*` listed in package.json.

Alternatively we could list all type defs here or enable `skipLibCheck` which does the same thing as `types: []` 😕

Reply:
to be more clear, something like:

```diff
- "types": ["jest"],
+ "types": ["gzip-js", "jest", "react", "react-dom"],
```
Might be ideal, but would require every typedef to be listed here manually 😢